lib/matplotlib/pyplot.py

```diff
@@ -2439,622 +2439,799 @@ def spy(Z, precision=0, marker=None, markersize=None, aspect='equal', **kwargs):
 
 # Autogenerated by boilerplate.py.  Do not edit as changes will be lost.
 @_autogen_docstring(Axes.acorr)
-def acorr(x, data=None, **kwargs):
-    ax = gca()
-    ret = ax.acorr(x, data=data, **kwargs)
+def acorr(
+        x, data=None, **kwargs
```

Comment:
Is there a good reason to have all these newlines? I think they make the code more unreadable.

Reply:
Got rid of the ones after the argument lists.  The ones before the argument list are present to make code generation simpler (we can just always indent by 8); note that previously the calls had sometimes "weird" indentations.